src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -671,9 +673,11 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
     // Arguments:
     // - sender: not used
     // - args: event data
-    void TermControl::_MouseClickHandler(Windows::Foundation::IInspectable const& /*sender*/,
+    void TermControl::_PointerPressedHandler(Windows::Foundation::IInspectable const& sender,
                                          Input::PointerRoutedEventArgs const& args)

```

Comment:
nit: does clang-format really not care that these args aren't aligned anymore? I thought it would...

Reply:
They're aligned. I think this might be a GitHub preview bug.

---
In reply to: [293539451](https://github.com/microsoft/terminal/pull/1248#discussion_r293539451) [](ancestors = 293539451)